designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/DeploymentManager.scala

```diff
@@ -113,14 +118,3 @@ trait DeploymentManager extends AutoCloseable {
   ): Future[SavepointResult]
 
 }
-
-trait AlwaysFreshProcessState { self: DeploymentManager =>
```

Comment:
We need a migration guide entry, that this class disappear and the `getProcessStates` signature changed. Let's bundle it with my MG entry

Reply:
The content is fine but I think that we misunderstand the "bundle" part. I mean: 
`DeploymentManager` related changes:
 -...
 - ...
As a developer I would like to get a summary what changed in the interface instead of reading each entry separated.